extensions/wikia/WallNotifications/GlobalNavigationWallNotificationsController.class.php

```diff
@@ -1,305 +1,54 @@
 <?php
+
 /*
  * in here we render Notifications in top-right corner of Wikia interface
  */
 
 
-class GlobalNavigationWallNotificationsController extends WikiaController {
+class GlobalNavigationWallNotificationsController extends WallNotificationControllerBase {
 
 	const NOTIFICATION_TITLE_LIMIT = 48;
 
 	private $wallHelper;
 
 	public function __construct() {
+		parent::__construct();
 		$this->app = F::App();
 	}
 
 	public function Index() {
```

Comment:
There is no valid reason at all.
It was my sloppy mistake that show us that we cannot trust in tests.
Thank you for pointing that